src/node/internal/internal_utils.ts

```diff
@@ -28,7 +28,7 @@
 
 /* eslint-disable */
 
-export function normalizeEncoding(enc: string) {
+export function normalizeEncoding(enc?: string) {
```

Comment:
let's add return type since it gets more complicated? `: string | undefined` should do.